packages/slate-react/src/hooks/use-slate-selector.tsx

```diff
@@ -113,16 +113,23 @@ export function useSlateSelector<T>(
  * Create selector context with editor updating on every editor change
  */
 export function getSelectorContext(editor: Editor) {
```

Comment:
This is only used in one place. It's probably worth renaming it to `useSelectorContext` so we don't have to disable the hooks rule for a bunch of hook calls.